src/System.Private.CoreLib/Resources/Strings.resx

```diff
@@ -2861,7 +2861,7 @@
     <value>Cannot create arrays of ByRef-like values.</value>
   </data>
   <data name="NotSupported_ByRefToByRefLikeReturn" xml:space="preserve">
-    <value>ByRef to ByRefLike return values not supported in reflection invocation.</value>
+    <value>ByRef to ByRef-like return values not supported in reflection invocation.</value>
```

Comment:
Nit: "values not supported" => "values are not supported"

Reply:
Or maybe "Returning ByRef-like types by reference is not supported in reflection invocation."?